collections/_priorities/macos-00-improve-behavior-of-multiple-monitors-with-different-resolutions.md

```diff
@@ -0,0 +1,16 @@
+---
+last_updated: "2024-11-30"
+category: macos
+rank: 0
+created: "2024-11-30"
+stage:
+  # Must be dates (YYYY-MM-DD), empty if not there yet.
+  planning: ""
+  ongoing: ""
+  done: ""
+anchor: "improve-behavior-of-multiple-monitors-with-different-resolutions"
+title: "Improve behavior of multiple monitors with different resolutions"
+description: |
+  Godot games should be able to seamlessly work when dragged across multiple monitors even when the monitors have different resolutions.
```

Comment:
```suggestion
  Godot games should be able to seamlessly work when dragged across multiple monitors even when the monitors have different resolutions and DPI scaling factors.
```

Reply:
Also, is not macOS specific, and should be in `Platforms` since it's relevant for all desktops.

Reply:
I removed it from macOS and put it in Platforms.